packages/console-extension/src/index.ts

```diff
@@ -399,6 +402,17 @@ function activateConsole(app: JupyterLab, mainMenu: IMainMenu, palette: ICommand
     isEnabled
   });
 
+  commands.addCommand(CommandIDs.toggleEcho, {
+    label: 'Toggle echo…',
```

Comment:
We reserve the ellipsis for commands that require further confirmation, like a dialog box. Since this doesn't require further confirmation, we shouldn't have the ellipsis here.